### DISCUSSION TITLE
How to change MaterialDesign PopupBox content background color?

### DISCUSSION THREAD

**Discussion by alexkeoh** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3051] [Q&A]
I try to change the popup content background to navy blue color, but notice the top and bottom line came with white color. 
would like to ask the guidelines for changing popupbox background color.

![image](https://user-images.githubusercontent.com/39609677/213130630-fbf4242a-816d-4c59-97fb-2d40de7f6ffe.png)

**Comment by nicolaihenriksen** [comment]
@alexkeoh I guess it all depends on what you're trying to achieve.

Generally you can see it this way: There are 2 overall "themes" ([Light and Dark](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/e02fd04df54428c862c777d6bd2f2acd0dc7d1fc/MaterialDesignThemes.Wpf/BaseTheme.cs#L3-L8); `Inherit` will use the one you have defined in your Windows settings). Within each theme you can then set a [Primary and a Secondary](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/e02fd04df54428c862c777d6bd2f2acd0dc7d1fc/MaterialDesignColors.Wpf/MaterialDesignColor.cs#L3-L44) color which will affect the controls.

You can of course, as you have done, override this by setting `Background`/`Foreground` properties directly, but then you will need to deal with a lot of changes most likely; depending on where you apply the custom color.

Perhaps a good starting point is to start the demo tool like you have, and then navigate to the Color Tool. In there you can play around with the colors to see if you can find something that you find appropriate. When "Color Adjustment" is enabled, you may not get the exact color you pick, that is because a contrast-algorithm may determine that using the selected color will not provide enough contrast for things to be readable and thus it compensates by selecting a color "nearby" which does provide enough contrast.

![image](https://user-images.githubusercontent.com/19572699/213172812-00a47f4f-9053-439d-a4ff-16c352f1db8d.png)

![image](https://user-images.githubusercontent.com/19572699/213172190-a8968404-d6d7-4f02-b385-47a1f2d08a5c.png)
